Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md

```diff
@@ -0,0 +1,228 @@
+- hosts: laba5hosts
+  tasks:
+    - name: Print facts
+      debug:
+        msg:
+          - Hostname: "{{ ansible_hostname }}"
+          - OS/ver: "{{ ansible_distribution }}/{{ ansible_distribution_version }}"
+          - Mount point: "{{ ansible_mounts[0].mount }}"
+          - Mount point capasity: "{{ ansible_mounts[0].size_total }}"
+          - Mount point size avalible: "{{ ansible_mounts[0].size_available }}" 
+          - RAM capasity: "{{ hostvars[inventory_hostname].ansible_memory_mb }}"
+          - RAM free: "{{ hostvars[inventory_hostname].ansible_memfree_mb }}"
+
+```
+
+### The results
+
+Run the following command from the folder 05.Ansible.Run.
+
+```bash
+
+➜  05.Ansible.Run ansible-playbook ansiblerunprint.yaml -i inventory/main.yaml --ask-vault-pass
+Vault password: 
+
+PLAY [laba5hosts] **************************************************************
+
+TASK [Gathering Facts] *********************************************************
+Thursday 27 June 2019  07:26:26 +0200 (0:00:00.241)       0:00:00.241 ********* 
+ok: [sa_cent]
+
+TASK [Print facts] *************************************************************
+Thursday 27 June 2019  07:27:13 +0200 (0:00:46.930)       0:00:47.172 ********* 
+ok: [sa_cent] => {
+    "msg": [
+        {
+            "Hostname": "sa-cent"
+        }, 
+        {
+            "OS/ver": "CentOS/7"
+        }, 
+        {
+            "Mount point": "/boot"
+        }, 
+        {
+            "Mount point capasity": "1063256064"
+        }, 
+        {
+            "Mount point size avalible": "796987392"
+        }, 
+        {
+            "RAM capasity": {
+                "nocache": {
+                    "free": 769, 
+                    "used": 221
+                }, 
+                "real": {
+                    "free": 286, 
+                    "total": 990, 
+                    "used": 704
+                }, 
+                "swap": {
+                    "cached": 0, 
+                    "free": 2047, 
+                    "total": 2047, 
+                    "used": 0
+                }
+            }
+        }, 
+        {
+            "RAM free": "286"
+        }
+    ]
+}
+
+PLAY RECAP *********************************************************************
+sa_cent                    : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+
+Thursday 27 June 2019  07:27:13 +0200 (0:00:00.030)       0:00:47.202 ********* 
+=============================================================================== 
+Gathering Facts -------------------------------------------------------- 46.93s
+Print facts ------------------------------------------------------------- 0.03s
+➜  05.Ansible.Run 
+
+```
+
+## Part 2 - Playbook To Play With The Ansible User: 
+
+Create playbook ansiblerunuser.yaml in the folder 05.Ansible.Run as follows:
+
+```bash
+
+---
+ - hosts: laba5hosts
+   become: yes 
+   gather_facts: no
+   vars:
+    - user_pass: 'Qwerty1'
```

Comment:
Why is that doing here, ansible vault!!!

Reply:
Or should be provided like extra var

Reply:
Fixed